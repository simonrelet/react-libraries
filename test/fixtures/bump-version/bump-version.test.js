'use strict'

const fs = require('fs-extra')
const path = require('path')
const { callScriptInPackage } = require('../../scriptExecution')

const PACKAGE_JSON_PATH = path.join(__dirname, 'package.json')
const PACKAGE_JSON = {
  name: 'bump-version',
  version: '1.0.0',
  description: 'Description of the library.',
  repository: {
    url: 'https://repository.url',
  },
}

const NEW_VERSION = '2.0.0'

describe('bump-version', () => {
  // beforeEach(async () => {
  //   await fs.writeJSON(PACKAGE_JSON_PATH, PACKAGE_JSON, { spaces: 2 })
  // })
  // afterEach(async () => {
  //   await fs.remove(PACKAGE_JSON_PATH)
  // })
  // it('should not fail', async () => {
  //   const result = await callScriptInPackage(__dirname, 'bump-version', [
  //     NEW_VERSION,
  //   ])
  //   expect(result).toMatchSnapshot()
  // })
  // it('should bump the new version', async () => {
  //   const result = await callScriptInPackage(__dirname, 'bump-version', [
  //     version,
  //   ])
  //   expect(result).toMatchSnapshot()
  // })
})
