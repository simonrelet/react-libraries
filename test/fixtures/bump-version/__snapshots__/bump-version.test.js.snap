// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`bump-version should not fail 1`] = `
Exit code: 0

--- Output ---
package.json -> package.json
--------------
`;
